src/main/java/com/google/cloud/discotoproto3converter/DiscoToProto3ConverterApp.java

```diff
@@ -49,16 +54,22 @@ public static void main(String[] args) throws IOException {
     converter.convert(
         parsedArgs.get("--discovery_doc_path"),
         parsedArgs.get("--output_root_path"),
-        parsedArgs.get("--output_file_name"));
+        parsedArgs.get("--output_file_name"),
+        parsedArgs.get("--service_ignorelist"),
```

Comment:
We should add a brief mention to the design doc about why we need to ignore messages and services.
@vam-google